### ISSUE TITLE
v0.23.0rc2 Final Download does't work

### ISSUE THREAD

**Issue by abi-aryan** [pandas-dev/pandas#21024]
#### Code Sample, a copy-pastable example if possible

```python
# Your code here
conda install -c conda-forge --label=rc pandas=0.23.0rc2

gives 

usage: conda [-h] [-V] command ...
conda: error: unrecognized arguments: --label=rc

error.

```
#### Problem description

[this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]

**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates!

**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.

For documentation-related issues, you can check the latest versions of the docs on `master` here:

https://pandas-docs.github.io/pandas-docs-travis/

If the issue has not been resolved there, go ahead and file it in the issue tracker.

#### Output of ``pd.show_versions()``

<details>

[paste the output of ``pd.show_versions()`` here below this line]

</details>

**Comment by jorisvandenbossche** [comment]
The command to install the release candidate is:

```
conda install -c conda-forge/label/rc pandas=0.23.0rc2
```

Did you find the install command you used somewhere online? (so in that case we should correct that)

**Comment by abi-aryan** [comment]
Yup @jorisvandenbossche on the [homepage](https://pandas.pydata.org/) 

![screen shot 2018-05-14 at 02 12 43](https://user-images.githubusercontent.com/7281944/39988560-8387a75e-571c-11e8-9bc7-aec9a1397b70.png)

**Comment by jorisvandenbossche** [comment]
Ah, thanks a lot for mentioning this, will fix that! (in the announcement mail on the mailing list it was stated correctly)
